synapse/cmds/cortex.py

```diff
@@ -210,6 +217,13 @@ class StormCmd(s_cli.Cmd):
         --hide-tags: Do not print tags.
         --hide-props: Do not print secondary properties.
         --hide-unknown: Do not print messages which do not have known handlers.
+        --show-nodeedits:  Show full nodeedits (otherwise printed as a single .)
+        --editformat <format>: What format of edits the server shall emit.
+                Options are
+                   * nodeedits (default),
+                   * splices (similar to < 0.2.0),
+                   * count (just counts of nodeedits), or
```

Comment:
i dont think this needs comma/or when it's a bullet list

Reply:
It does if you're writing in prose.